src/org/ods/component/Pipeline.groovy

```diff
@@ -370,10 +370,21 @@ class Pipeline implements Serializable {
 
   private void amendProjectAndComponentFromOrigin(Map config) {
     def block = {
-      def origin = script.sh(
-        script: 'git config --get remote.origin.url',
-        returnStdout: true
-      ).trim()
+      def origin
+      try {
+        origin = script.sh(
+          script: 'git config --get remote.origin.url',
+          returnStdout: true
+        ).trim()
+      } catch (err) {
+        script.dir ("../workspace@script") {
+          origin = script.sh(
+            script: 'git config --get remote.origin.url',
+            returnStdout: true
+          ).trim()
+        }
+      }
```

Comment:
Makes me wonder if we went down a dead-end and it would be better to configure it explicitly. Or maybe there are better ways to extract it (e.g. from OpenShift resources)?

Reply:
hmmmm, dude! ... assuming I have an openshift service at hand, I could get it via OC :) let me play with this

Reply:
I just remembered that I commented a while back - in different context - how to auto-configure the project ID: https://github.com/opendevstack/ods-quickstarters/issues/42#issuecomment-558631490

Reply:
@michaelsauter  - take a look now :)